README.md

```diff
@@ -34,30 +38,39 @@ The server component to the TourTrak system built using Django.
 * Setup the postGIS functions:
 
 ```
-  psql -d DCS -f /usr/share/postgresql/9.1/contrib/postgis-1.5/postgis.sql
-  psql -d DCS -f /usr/share/postgresql/9.1/contrib/postgis-1.5/spatial_ref_sys.sql
-  
+  psql DCS
+  CREATE EXTENSION postgis;
+  CREATE EXTENSION postgis-topology;
+  \q
 ```
 
 ## Postgres Database Commands
+<i>(This is not part of the install. Just advice)</i>
 #####Connecting to the database after setup
-1. Type `su postgres`
-2. Type `psql`
-3. Type `\connect DCS`
+1. Type `sudo -iu postgres`
+2. Type `psql DCS`
 
 #####Some helpful commands include:
 1. `\dt` to list all tables
 2. `\q` to quit
-3. `psql -d DCS -f /path/to/script/script.sql` for running scripts without having to connect
+3. `psql DCS < /path/to/script/script.sql` for running scripts without having to connect
 
 
-##Installation
+##Install The Application
 
-* Clone this repository into /usr/local/
-* <b><font color="red">IMPORTANT:</font> Rename the directory from devcycle-server to devcycle. Django does not support hyphen names for it's applications.</b>
-* Inside the root directory of your application, install all project dependencies by runnning our setup script `bash setup.sh`
-* Create a virtual host & WSGI file for the Apache server to display the Django application. Open the httpd.conf file 'nano /etc/apache2/httpd.conf'. Copy and paste the following, you may edit 
-these values if desired (such as where to collect static files).
+1. Install git
+	<br>`sudo apt-get install git`
+2. Create a 'devcycle' directory in /usr/local/
+	<br>`sudo mkdir /usr/local/devcycle`
+3. Clone this repository into /usr/local/devcycle
+ 	<br>`sudo git clone https://github.com/tourtrak/devcycle-server.git /usr/local/devcycle`
+4. <b><font color="red">IMPORTANT:</font> The directory must be named 'devcycle' and not 'devcycle-server'. Django does not support hyphen names for it's applications.</b>
+5. Inside the root directory of your application, install all project dependencies by running our setup script
+	<br>`cd /usr/local/devcycle`
+	<br>`sudo bash setup.sh`
```

Comment:
It might be worth noting that this script requires user input. That's something we can address later on because there are ways to run apt non-interactively so that it just goes ahead, but for now it's worth noting.